lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -483,6 +478,47 @@ def __init__(self, service: _ServiceType) -> None:
 
     def install(self) -> None:
         """Install Slurm using the `slurm` snap."""
+        self._init_ubuntu_hpc_ppa()
+        self._install_service()
+        # Debian package postinst hook does not create a `StateSaveLocation` directory
+        # so we make one here that is only r/w by owner.
+        _logger.debug("creating slurm statesavelocation directory")
```

Comment:
```suggestion
        _logger.debug("creating slurm StateSaveLocation directory")
```

Nit: we should use the "proper" name in logs.

Reply:
Tbh I was on the fence about keeping this entirely lowercase, but since you call it out, I'm fine with just quickly patching this and then merging the PR :smile: